src/org/labkey/test/tests/ExpTest.java

```diff
@@ -194,9 +189,7 @@ public void testSteps()
         // Wait for query to load
         waitForText("edit metadata");
         clickAndWait(Locator.linkWithText("edit metadata"));
-        waitForElement(Locator.xpath("//span[contains(text(), 'Reset to Default')]"), defaultWaitForPage);
-        click(Locator.xpath("//span").append(Locator.lkButton("Reset to Default")));
-        click(Locator.xpath("//span").append(Locator.lkButton("OK")));
-        waitForText(WAIT_FOR_JAVASCRIPT, "Reset successful");
+        designerPage = new DomainDesignerPage(getDriver());
+        designerPage.click(Locator.button("Reset To Default"));
```

Comment:
does the updated page button click for "Reset To Default" not have a confirmation dialog? looks like the old test was clicking that button and then clicking OK to confirm it.

Reply:
I think the confirmation for Reset to Default comes if there are any changes to the domain form in the current state. Do we always want the confirmation?

Reply:
I would say that, yes we do want the confirmation always. It could be the case that they had saved some metadata changes, then come back to this page and chose to reset at that point.

Reply:
changes related to this in the commit here: https://github.com/LabKey/platform/pull/949/commits/b9ce7eec0f41baaa13d9e8142334c0cc8d70a366